CHANGELOG.md

```diff
@@ -848,6 +849,7 @@
 [8105]: https://github.com/enso-org/enso/pull/8105
 [8150]: https://github.com/enso-org/enso/pull/8150
 [8235]: https://github.com/enso-org/enso/pull/8235
+[8307]: https://github.com/enso-org/enso/pull/8307
```

Comment:
How comes that a PR like this doesn't have any overview at all?

![obrazek](https://github.com/enso-org/enso/assets/26887752/08e0d8a0-2e13-491b-a86c-cf61a61b285d)

No reference to any issue!? No summary? No explanation why chosen path is the right one?